Userland/Applications/CalendarSettings/CalendarSettingsWidget.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022-2022, Olivier De Cannière <olivier.decanniere96@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "CalendarSettingsWidget.h"
+#include <Applications/CalendarSettings/CalendarSettingsWidgetGML.h>
+#include <LibConfig/Client.h>
+#include <LibGUI/ItemListModel.h>
+
+void CalendarSettingsWidget::apply_settings()
+{
+    m_first_day_of_week = m_first_day_of_week_combobox->text();
+    Config::write_string("Calendar"sv, "view"sv, "first_day_of_week"sv, m_first_day_of_week);
```

Comment:
Please capitalize config keys, to keep them consistent with the rest of the system. "first_day_of_week" => "FirstDayOfWeek", etc.